packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -5,6 +5,7 @@ import {
   presenceAvailableRegular,
   presenceAwayFilled,
   presenceBusyFilled,
+  presenceBlockedRegular,
```

Comment:
_nit:_ Keep alphabetical order

```suggestion
  presenceBlockedRegular,
  presenceBusyFilled,
```